### ISSUE TITLE
Want just enquiry

### ISSUE THREAD

**Issue by parikshitjadye** [microweber/microweber#605]
Can I make the website for multiple users by creating multiple user ids

**Comment by peter-mw** [comment]
yes you can make, there is module for user registration, just drop it on the page

**Comment by parikshitjadye** [comment]
Thank you for the response but please provide us the detailed explanation as I am new to laravel.

**Comment by parikshitjadye** [comment]
Beacuse at installation time it is asking for DB credentials, so is it creating new db to every user every time?

**Comment by peter-mw** [comment]
hi, if you want to create multi domain installation you can do it from command line, see this page 
http://docs.microweber.com/guides/installation_cli.md#multi-domain-scripted-installation

**Comment by parikshitjadye** [comment]
Hi Peter,
With Reference from I checked the documentation, but I want to create only user registration and login and on login user will be able to change its website.

Is this flow is available in current flow of project. please let us know what I need to do if flow is not available.

**Comment by peter-mw** [comment]
hi @parikshitjadye we have this flow available with combination of WHMCS+CPanel , please contact us on the [Slack chat](https://join.slack.com/t/microweber/shared_invite/enQtNDAxNTUxNjkzMjAxLWUzMWUyMDA0YTM3ODlhMmI4YTJiZDNiNzA1YzQxZjFkODM5MDQzZmEyMGM5NjI3ZDQ0YTA1Y2U2YjBmZDBkZDg) to explain how to setup

**Comment by parikshitjadye** [comment]
hii peter I have just sent you request on slack chat.
